pages/accounts/base.py

```diff
@@ -12,7 +12,7 @@ class AccountsBase(Page):
     """Base class."""
 
     URL_TEMPLATE = 'https://accounts{0}.openstax.org'.format(
-        os.getenv('instance', ''))
```

Comment:
Using env_var's here is probably a bad idea. This would be a PageObject being way too involved in implementation details.  In `cnx-automation` we have created fixtures that return the proper url. This is also suggested in the [pytest-base-url](https://pypi.org/project/pytest-base-url/) docs. This is what we do in cnx-automation https://github.com/openstax/cnx-automation/blob/master/tests/conftest.py#L47-L68

Reply:
yeah...it was a bit of a hack because I forgot to bring in the environment template file...